### PR TITLE
add droplet2phot to producer template

### PR DESCRIPTION
**Pull request by vespos**
Add entry to the producer template for the droplet2Photons workflow.

### CODE REVIEW

File: producers/smalldata_producer_template.py [hunk 1]
```diff
@@ -112,21 +112,34 @@ def getDropletParams(run):
...
         d2p_dict = {}
-        d2p_dict['threshold'] = 15
-        d2p_dict['mask'] = None
-        d2p_dict['aduspphot'] = 162
-        d2p_dict['offset'] = 81
-        d2p_dict['nData'] = 1e5 # int or None: sparsify arg
-        ret_dict['epix_alc'] = d2p_dict
+        # droplet args
+        d2p_dict['droplet'] = {
+            'theshold': 10,
```
Comment (silkenelson) [inline producers/smalldata_producer_template.py:128]: typo, missing 'r'

File: producers/smalldata_producer_template.py [hunk 2]
```diff
@@ -259,22 +272,31 @@ def define_dets(run):
                 det.addFunc(photonFunc(**phot[detname]))
             # Droplet algo
             if detname in drop:
-                if nData in drop:
-                    nData = drop.pop('nData')
+                if nData in drop[detame]:
```
Comment (silkenelson) [inline producers/smalldata_producer_template.py:277]: detame -> detname

**Review by silkenelson — COMMENTED** [review]
Approved w/ fixing of the two typos (unless I'm seeing things wrong...)

File: producers/smalldata_producer_template.py [hunk 3]
```diff
@@ -249,40 +261,55 @@ def define_dets(run):
...
+            
+            # psana photon count
             if detname in phot:
                 det.addFunc(photonFunc(**phot[detname]))
+            
             # Droplet algo
             if detname in drop:
-                if nData in drop:
-                    nData = drop.pop('nData')
+                if nData in drop[detame]:
```
Comment (vespos) [inline producers/smalldata_producer_template.py:277]:
```suggestion
                if nData in drop[detname]:
```

File: producers/smalldata_producer_template.py [hunk 4]
```diff
@@ -112,21 +112,33 @@ def getDropletParams(run):
...
         d2p_dict = {}
-        d2p_dict['threshold'] = 15
-        d2p_dict['mask'] = None
-        d2p_dict['aduspphot'] = 162
-        d2p_dict['offset'] = 81
-        d2p_dict['nData'] = 1e5 # int or None: sparsify arg
-        ret_dict['epix_alc'] = d2p_dict
+        # droplet args
+        d2p_dict['droplet'] = {
+            'theshold': 10,
```
Comment (vespos) [inline producers/smalldata_producer_template.py:128]:
```suggestion
            'threshold': 10,
```

File: producers/smalldata_producer_template.py (hunk 2 above)
Comment (vespos) [reply producers/smalldata_producer_template.py]: Fixed

File: producers/smalldata_producer_template.py (hunk 1 above)
Comment (vespos) [reply producers/smalldata_producer_template.py]: Fixed

**Review by silkenelson — APPROVED** [review]

**Pull request merged**